sdks/python/apache_beam/runners/portability/spark_runner.py

```diff
@@ -93,9 +93,7 @@ def path_to_jar(self):
       return self._jar
     else:
       if self._spark_version == '2':
```

Comment:
Why we still need this check?

Reply:
yes, if somebody explicitly requires spark 2 it should fail fast